package.json

```diff
@@ -58,6 +60,7 @@
     "postcss": "^8.4.27",
     "postcss-loader": "^7.3.3",
     "postcss-scss": "^4.0.6",
+    "regenerator-runtime": "^0.14.0",
```

Comment:
to match the Webpack Encore Bundle recipe, see https://github.com/symfony/recipes/blob/main/symfony/webpack-encore-bundle/1.10/package.json#L9